File: book.php
<html>
<head>
    <title>SQL SELECT Statement</title>
</head>
<body>
<table align="center" border="1">
<?php
    $cnx = new mysqli('localhost', 'root', 'vwxyz', 'demo');

    if ($cnx->connect_error)
        die('Connection failed: ' . $cnx->connect_error);

    $query = 'SELECT * FROM book';
    $cursor = $cnx->query($query);
    while ($row = $cursor->fetch_assoc()) {
        echo '<tr>';
        echo '<td>' . $row['isbn'] . '</td><td>' . $row['title'] . '</td><td align="right">' . $row['price'] .'</td>';
        echo '</tr>';
    }

    $cnx->close();
?>
</table>
</body>
</html>

File: book.py
import mysql.connector


def insert(cursor):
    query = 'INSERT INTO book(isbn,title,price) VALUES (%s,%s,%s)'
    cursor.execute(query, ('0385514239','Origin',2995))


def update(cursor):
    query = 'UPDATE book SET price=%s WHERE isbn=%s'
    cursor.execute(query, (29.95,'0385514239'))


try:
    cnx = mysql.connector.connect(host='localhost', user='root', password='vwxyz', database='demo')
    cursor = cnx.cursor()

    insert(cursor)
    cnx.commit()

    update(cursor)
    cnx.commit()

    cursor.close()
except mysql.connector.Error as err:
    print(err)
finally:
    try:
        cnx
    except NameError:
        pass
    else:
        cnx.close()